src/index.js

```diff
@@ -505,116 +526,67 @@ export var h = function(name, props) {
     : createVNode(name, props, children, null, props.key, DEFAULT)
 }
 
-var resolveNode = function(newNode, oldNode) {
-  var node = newNode
-
-  if (node.type === LAZY_NODE) {
-    node =
-      oldNode && oldNode.lazy && isSameValue(node.lazy, oldNode.lazy)
-        ? oldNode
-        : node.render()
-  }
-  return node
-}
-
-export var Lazy = function(props) {
-  return {
-    type: LAZY_NODE,
-    key: props.key,
-    lazy: props,
-    render: function() {
-      var node = props.render(props)
-      node.lazy = props
-      return node
-    }
-  }
-}
-
-var cancel = function(sub) {
-  sub[2]()
+var isSameAction = function(a, b) {
+  return isArray(a) && isArray(b) && typeof a[0] === "function" && a[0] === b[0]
 }
 
-var isSameValue = function(a, b) {
```

Comment:
@jorgebucaran This function was used inside of `resolveNode`, so the current alpha release is broken.

Reply:
@SkaterDad Thanks, I'll fix that.